### ISSUE TITLE
Warning: Failed prop type: Calling PropTypes validators directly is not supported by the `prop-types` package

### ISSUE THREAD

**Issue by jankarres** [keppelen/react-facebook-login#141]
Example code:

```
import * as React from "react";
import { render } from "react-dom";
import FacebookLogin from "react-facebook-login";

render((
          <FacebookLogin
            appId="583141595213073"
            fields="public_profile,email,user_birthday,user_friends,user_likes"
            callback={this.responseFacebook} />
),
  document.querySelector("#app")
);
```

Warning in console appears:

`Warning: Failed prop type: Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types`

Used version (current at npm): 3.6.0

**Comment by qaiser110** [comment]
I'm getting the same error

**Comment by mgreer** [comment]
I'm also seeing it, but don't see the issue in the code. All of the prop-types seem to be called from PropTypes, and I don't see any third party PropTypes. Could be in a dependency?

**Comment by kidchenko** [comment]
+1 on this error, well, I was seeing the `packages.json` of this package but the obly depency is from react. Maybe can be a devDependencies?

**Comment by ofergoli** [comment]
Also getting the same error, any ideas what's caused it?

**Comment by lrojas94** [comment]
+1, I see the error too.. I checked the code and it seems alright... could it be related to `prop-types` itself maybe?

**Comment by lrojas94** [comment]
Seems to be a false positive. To fix, simply update to `prop-types@15.5.10`:
https://github.com/facebook/prop-types/issues/38

**Comment by ofergoli** [comment]
Tnx @lrojas94 ,fixed!
